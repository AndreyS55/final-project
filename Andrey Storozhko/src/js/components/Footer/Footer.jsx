import React from 'react';
import Button from '@material-ui/core/Button';
import styles from './Footer.scss';

const Footer = () => (
  <footer className={styles.pokedexFooter}>
    <div className={styles.footerWrapper}>
      <div className={styles.footerText}>This App was made for EPAM Training</div>
      <Button
        href='https://github.com/AndreyS55/Pokedex'
        size="large"
        color="default"
        variant="contained"
      >
        GitHub
      </Button>
    </div>
  </footer>
);

export default Footer;
