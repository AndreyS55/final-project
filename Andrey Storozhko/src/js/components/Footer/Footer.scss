@import '../../../styles/environment/colors';
@import '../../../styles/environment/mixins';

.pokedexFooter {
  flex: 0 0 auto;
  height: 7vh;
  width: 100%;
  background-color: $wht;
  border-top: 1px solid $gry;
}

.footerWrapper {
  width: 1200px;
  height: 100%;
  margin: 0 auto;
  display: flex;
  justify-content: space-evenly;
  align-items: center;

  @include desktop() {
    width: 900px;
  }

  @include laptop() {
    width: 750px;
  }

  @include tablets() {
    width: 100%;
  }

  @include phones() {
    width: 100%;
  }
}

.footerText {
  @include phones() {
    font-size: 0.75rem;
  }
}
