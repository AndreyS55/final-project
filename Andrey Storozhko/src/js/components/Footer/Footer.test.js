import React from 'react';
import Footer from './Footer';

it('should render a Footer', () => {
  const wrapper = shallow(<Footer />);
  expect(wrapper).toMatchSnapshot();
});
