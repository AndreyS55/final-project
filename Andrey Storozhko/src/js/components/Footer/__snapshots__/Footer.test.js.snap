// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a Footer 1`] = `
<footer
  className="pokedexFooter"
>
  <div
    className="footerWrapper"
  >
    <div
      className="footerText"
    >
      This App was made for EPAM Training
    </div>
    <WithStyles(Button)
      color="default"
      href="https://github.com/AndreyS55/Pokedex"
      size="small"
      variant="contained"
    >
      GitHub
    </WithStyles(Button)>
  </div>
</footer>
`;
